tee-worker/core-primitives/stf-executor/src/executor.rs

```diff
@@ -139,7 +139,7 @@ where
 			self.node_metadata_repo.clone(),
 		) {
 			Err(e) => {
-				if let Err(e) = self.ocall_api.update_metric(EnclaveMetric::FailedTrustedOperationIncrement) {
+				if let Err(e) = self.ocall_api.update_metric(EnclaveMetric::FailedTrustedOperationIncrement(trusted_call.call.clone())) {
```

Comment:
maybe you can pass call name here instead of cloning the whole `trusted_call` ?

Reply:
Actually, I was thinking of extending it to use it like an indexer for trusted_call, that is the metric would be something like this 
```
(Block Number, Trusted Operation, Caller) 
```
And I think the trusted_call has those things, but yeah I can still optimize it to pass things that i need 
Created an initial dashboard that looks like this: 
![Screenshot 2023-09-08 at 6 34 20 PM](https://github.com/litentry/litentry-parachain/assets/42486737/de85c9b9-341e-4ebf-9a32-ce77c1aee263)

Reply:
If the implementation is going to need it anyway then there is no need to remove it now in order to introduce it later 🙃